tests/Unit/ControlSystem/ControlErrors/Test_Shape.cpp

```diff
@@ -64,10 +65,10 @@ void test_shape_control_error() {
   MAKE_GENERATOR(generator);
   domain::FunctionsOfTime::register_derived_with_charm();
 
-  const std::array<double, 3> origin{{0.0, 0.0, 0.0}};
+  const tnsr::I<double, 3, Frame::Grid> origin{0.};
```

Comment:
For consistency have `0.` be `0.0` to match the formatting elsewhere in this file